app/models/board.rb

```diff
@@ -14,10 +14,18 @@ def to_param
     slug
   end
 
-  def member?(user, role = %w[member creator])
+  def member?(user, role = %w[member creator admin host])
```

Comment:
I'm not sure we really need these methods to be in this model. Because they are related exclusively to policies